libs/ui-lib-tests/cypress/support/interceptors.ts

```diff
@@ -225,7 +253,36 @@ const addAdditionalIntercepts = () => {
   cy.intercept('GET', '/api/assisted-install/v2/default-config', defaultConfig);
 };
 
-const loadAiAPIIntercepts = (conf: AIInterceptsConfig | null) => {
+const addCustomManifestsIntercepts = (loadManifestContent: boolean | false) => {
+  cy.intercept('GET', `${clusterApiPath}/manifests`, mockCustomManifestResponse).as('manifests');
+  cy.intercept('PATCH', `${clusterApiPath}/manifests`, mockCustomManifestResponse);
+  cy.intercept('DELETE', `${clusterApiPath}/manifests`);
+  if (loadManifestContent) {
+    cy.intercept(
+      'GET',
+      `${clusterApiPath}/manifests/files?folder=manifests&file_name=manifest1.yaml`,
+      mockCustomManifestFileResponse,
+    );
```

Comment:
If I would like to craft my own `folder name` or `file name` during a test, I won't be able to reuse these intercepts...

Reply:
True. I'll implement in the future as a followup